base/uk.ac.stfc.isis.ibex.configserver/src/uk/ac/stfc/isis/ibex/configserver/editing/EditableConfiguration.java

```diff
@@ -381,17 +412,20 @@ public void removeIocs(List<EditableIoc> iocs) {
      * @return The editable IOCs associated with the configuration
```

Comment:
I think this comment is now incorrect, we're returning all IOCs, editable or not